### ISSUE TITLE
Suggestion for implementing improvements related to Expulsion Pylon

### ISSUE THREAD

**Issue by GnomoMuitoLoco** [jeremiahwinsley/pylons#25]
"Expulsion Pylon:
Intended for use on servers, this block allows you to expel other players from the chunk it's placed in. You can whitelist players with a Player Filter - OPs are always whitelisted."

This mod is helping me against theft in protected properties and players like to use it, but some team members have specific permission nodes and I can't give Op to everyone because there is a hierarchy, would have the possibility of implementing a permission to ignore players by nickname? or some specific permission, something like LuckPerms permissions?
By the way, your mod is very good and useful.

**Comment by jeremiahwinsley** [comment]
I haven't done much with permissions, do you have an example of what you're trying to do? The player filter allows you to ignore specific players.

**Comment by GnomoMuitoLoco** [comment]
Of course, a practical example, on my server there are some positions, the first of which is Members, Helpers, Moderators, and Adm, but only Moderators and Adm have OP, what I'm suggesting is that somehow, you could put player name instead of op permission. For example, it is not possible to place a filter with each Helper and Moderator on the basis of all players because the number of players is high, and not all staff members have op, so if there was a way to allow it by nick, for example, such a person can enter all lands because it has direct permission in the configuration, I don't know if it was clear, but this would be very good in case the Helper wants to go to the person to help (even though he doesn't have op or card) even to investigate, they can't access members' land but they should and in this part the mod ends up preventing it, so I thought maybe there was some permission that would authorize entry even without having OP.

**Comment by jeremiahwinsley** [comment]
I see, a permission to bypass all pylons like OP does. Yes, I think that sounds doable.
